src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -62,6 +63,7 @@ public JobHostConfiguration(string dashboardAndStorageConnectionString)
             }
 
             Singleton = new SingletonConfiguration();
+            Aggregator = new AggregatorConfiguration();
```

Comment:
I had that originally -- but then realized there's really no default logging configuration at the moment, so I was left with just the Aggregation config. I didn't think about putting the LoggingFactory on there, though (it's currently on the HostConfig itself). This is a good idea -- I can come back and clean that up in the next PR if you're okay with it.